homeassistant/components/airvisual/config_flow.py

```diff
@@ -48,6 +58,8 @@ def cloud_api_schema(self):
         """Import a config entry from configuration.yaml."""
         await self._async_set_unique_id(import_config[CONF_API_KEY])
 
+        await self._async_test_client_api_key(import_config[CONF_API_KEY])
```

Comment:
Can we pass the imported config to the user step instead after the modification below?